css/styles.css

```diff
@@ -665,4 +700,28 @@ Contains email and socials
     flex-direction: row;
     gap: 1%;
   }
+
+  .mobile-fields {
+    display: none;
+  }
+
+  input[class="desktop-fields"] {
+    width: 49%;
+    display: inline-flex;
+  }
+
+  form input:first-child {
+    background-color: transparent;
+  }
+
+  .form-button {
+    margin-left: 0;
+    width: 20%; /* TO FIX */
+    height: 100%;
+  }
```

Comment:
- [ ] As we are aiming for a design identicle to the figma design please 🙏 make sure the paddings in the input and button of your design also match with that of the figma. I suggest adding some padding top and reduce padding from the left of the `message` field. Also please reduce some padding top and bottom of the `name` and `email` field. And add padding top and bottom on the button. 👍 

Figma Design:
<img width="341" alt="Screenshot 2023-03-31 at 9 28 16 AM" src="https://user-images.githubusercontent.com/44798044/229027162-22681766-403b-4e2f-b5d0-c4130bb2aef8.png">

Your design:

<img width="750" alt="Screenshot 2023-03-31 at 9 28 20 AM" src="https://user-images.githubusercontent.com/44798044/229028139-92be76d7-8b79-4246-bdbb-151df1c1d0ce.png">

Reply:
Done